React/Modules/RCTServerUtils.m

```diff
@@ -0,0 +1,21 @@
+
+#import "RCTServerUtils.h"
+#import "RCTIPAddress.h"
+
+@implementation RCTServerUtils
+
++ (NSURL *)getServerURL:(NSString *)path
+{
+  NSUserDefaults *standardDefaults = [NSUserDefaults standardUserDefaults];
+  NSInteger port = [standardDefaults integerForKey:@"websocket-executor-port"] ?: 8081;
+
+#if TARGET_OS_SIMULATOR
```

Comment:
Inside `local-cli/start_packager.sh`, you want to use `$TARGET_OS_SIMULATOR` to save either `localhost` or the IP address into a file that will be loaded during runtime. Does that all sound right? Should it save the port in that file, too?